packages/mdc-ripple/foundation.js

```diff
@@ -365,48 +426,38 @@ class MDCRippleFoundation extends MDCFoundation {
     this.adapter_.computeBoundingRect();
   }
 
+  resetActivationState_() {
+    // Take note of previous activation event type to ignore follow-on event for the same interaction on touch devices
+    this.previousActivationEvent_ = this.activationState_.activationEvent;
+    this.activationState_ = this.defaultActivationState_();
+    setTimeout(() => this.previousActivationEvent_ = null, 100);
```

Comment:
Is the `setTimeout` to account for delays between when same-interaction "follow-on" events are triggered?

Reply:
Yup. Do you think my comment above these lines needs improvement?